tests/unittests/sources/test_configdrive.py

```diff
@@ -389,6 +389,7 @@ def setUp(self):
             M_PATH + "util.find_devs_with", "m_find_devs_with", return_value=[]
         )
         self.tmp = self.tmp_dir()
+        self.allowed_subp = True
```

Comment:
@rjschwei without this some tests failed, right?

Setting this allows these tests to call udevadm settle on the host. It would be better to avoid the udevadm dependency altogether by mocking out the new `util.udevadm_settle` call.

Reply:
Thanks for fixing this for me :D